packages/calypso-e2e/src/lib/blocks/block-flows/all-form-fields.ts

```diff
@@ -52,32 +59,43 @@ export class AllFormFieldsFlow implements BlockFlow {
 			[ 'Date Picker', 'Add label…' ],
 			[ 'Phone Number Field', 'Add label…' ],
 			[ 'Multi-line Text Field', 'Add label…' ],
-			[ 'Checkbox', 'Add label…' ],
+			[ 'Checkbox', isRefactor ? 'Add option…' : 'Add label…' ],
 			[ 'Multiple Choice (Checkbox)', 'Add label' ],
 			[ 'Single Choice (Radio)', 'Add label' ],
 			[ 'Dropdown Field', 'Add label' ],
 			[ 'Terms Consent', 'Add implicit consent message…' ],
 		];
 		for ( const [ blockName, accessibleLabelName ] of remainingBlocksToAdd ) {
-			await this.addFieldBlockToForm( context, blockName );
+			await this.addFieldBlockToForm( context, blockName, lastBlockName, isRefactor );
```

Comment:
See [this comment](https://github.com/Automattic/wp-calypso/pull/103974/files#r2127587828) below for updated logic in the AddFieldBlockToForm() method.